docs/guide/ops/catalog/index.md

```diff
@@ -0,0 +1,163 @@
+-->
+
+### Catalog items
+
+An item to be added to the catalog is defined in YAML. This looks like any other 
+YAML blueprint, but also has a `brooklyn.catalog` section giving appropriate metadata.
+
+An example is shown below.
+
+{% highlight yaml %}
+brooklyn.catalog:
+  id: org.example.MySQL
+  version: 1.0
+  iconUrl: classpath://mysql.png
+  description: MySql is an open source relational database management system (RDBMS)
+  libraries:
+    - url: http://example.com/path/to/my-dependency-1.2.3.jar
+    - url: http://example.com/path/to/my-other-dependency-4.5.6.jar
+
+services:
+- type: brooklyn.entity.database.mysql.MySqlNode
+{% endhighlight %}
+
+To explain the `brooklyn.catalog` section:
+
+- The `id: MySQL` section specifies a unique ID used by Brooklyn to identify the catalog item. 
+  Other blueprints can reference the catalog item using this id.
+- The `version: 1.0` section provides a unique version for the *blueprint*. NOTE: This is *not* the version of the software
+being installed (in this case MySQL).
+- The `iconUrl: classpath://...` is an optional, but recommended, section that allows you to provide the URL of a graphic to be 
+displayed in the 'Add Application' dialog alongside the blueprint name. If using a URL of the form `classpath://...`, then
+the image should be on Brooklyn's classpath (e.g. in the `conf` folder of the Brooklyn distro).
+- The `description: ...` section allows you to give a free-format description of the blueprint, which is displayed in the 
+'Add Application' dialog.
+
+The `libraries` section references OSGi bundles required for the blueprint. It can be omitted if everything
+required by the blueprint is already on the Brooklyn classpath. The URLs should be to stable OSGi bundles -
+if the bundle at this URL changes then this could impact applications if the Brooklyn server is restarted
+or fails over to a standby node.
+
+
+To reference a catalog item in another blueprint, simply use its id and optionally its version number.
+For example: 
+
+{% highlight yaml %}
+services:
+- type: org.example.MySQL:1.0
+{% endhighlight %}
+
+
+### Adding to the Catalog
+
+To add a catalog item to the catalog, post the YAML file to Brooklyn's REST API by using the `curl` command as
+follows (substituting your own usename:password, URL and file path):
+
+{% highlight bash %}
+curl -u admin:password http://127.0.0.1:8081/v1/catalog --data-binary @/path/to/mysql-catalog.yaml
+{% endhighlight %}
+
+
+
+### Deleting from the Catalog
+
+You can delete a versioned item from the catalog using the REST API. For example, to delete
+the item with id `org.example.MySQL` and version `1.0`:
+
+{% highlight bash %}
+curl -u admin:password -X DELETE http://127.0.0.1:8081/v1/catalog/entities/MySQL/1.0
+{% endhighlight %}
+
+**Note** Catalog items should not be deleted if there are running apps which were created using the same item. During
+rebinding the catalog item is used to reconstruct the entity.
```

Comment:
agree this would be nice, along with `force` flag to force deletion even if in use ("because i know what i'm doing")